code/game/objects/items/grenades/grenade.dm

```diff
@@ -102,8 +102,10 @@
 		return ..()
 
 /obj/item/grenade/attack_hand()
+	. = ..()
+	if(.)
+		return
 	walk(src, null, null)

```

Comment:
What the hell is even going on here. Just remove this.

Reply:
lol ok